.github/workflows/prioritize-by-reactions.yaml

```diff
@@ -0,0 +1,65 @@
+name: Prioritize Issues by Reactions
+
+on:
+  schedule:
+    - cron: '0 */4 * * *'   # Runs every 4 hours
+  workflow_dispatch:       # Allows manual runs
+
+jobs:
+  prioritize:
+    permissions:
+      issues: write
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Install jq
+        run: sudo apt-get install -y jq
+
+      - name: Prioritize issues by reactions
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          REPO: microsoft/vsmarketplace
+          HIGH: 10
+          MEDIUM: 5
+        run: |
+          issue_numbers=$(gh issue list -R "$REPO" --state open --json number -q '.[].number')
+
+          for ISSUE in $issue_numbers; do
+            echo "🔍 Checking Issue #$ISSUE"
+
+            reactions=$(curl -s \
+              -H "Accept: application/vnd.github.squirrel-girl-preview+json" \
+              -H "Authorization: token $GH_TOKEN" \
+              https://api.github.com/repos/$REPO/issues/$ISSUE/reactions)
+
+            total=$(echo "$reactions" | jq 'length')
+
+            if [ "$total" -ge "$HIGH" ]; then
+              priority="p/0"
```

Comment:
nit: can we just make these p0, p1 and p2? (No forward slash)

Reply:
Or '**Priority - 2**' to align with NuGetGallery [Priority - 2](https://github.com/NuGet/NuGetGallery/labels/Priority%20-%202)